### PR TITLE
test/typings, devdependencies

### PR DESCRIPTION
**Pull request by scottbommarito**
cleanup of automated testing

### CODE REVIEW

File: typings/codePush.d.ts [hunk 1]
```diff
@@ -1,400 +0,0 @@
-// Type definitions for Apache Cordova CodePush plugin.
```
Comment (scottbommarito) [inline typings/codePush.d.ts]: the following removed files are cordova plugin d.ts files that i accidentally checked in

**Comment by scottbommarito** [comment]
@lostintangent

**Comment by lostintangent** [comment]
Looks great!

**Comment by scottbommarito** [comment]
@geof90

File: .gitignore [hunk 2]
```diff
@@ -149,8 +148,5 @@ proguard/
 # Android Studio captures folder
 captures/
 
-# DELETE THIS AFTER PLUGIN TESTING FRAMEWORK IS ON NPM
-# ALSO SEE THE node_modules/* above
-!node_modules/
-!node_modules/code-push-plugin-testing-framework/
-!node_modules/code-push-plugin-testing-framework/*
\ No newline at end of file
+# REMOVE AFTER NPM
```
Comment (geof90) [inline .gitignore]: # Remove after this framework is published on NPM

**Comment by geof90** [comment]
LGTM

**Pull request merged**